Hazelcast.Net/Hazelcast.Config/ClientNetworkConfig.cs

```diff
@@ -133,56 +175,110 @@ public virtual bool IsSmartRouting()
             return _smartRouting;
         }
 
-        // required for spring module
+        /// <summary>
+        /// Sets all addresses
+        /// </summary>
+        /// <param name="addresses">list of addresses</param>
+        /// <returns><see cref="ClientNetworkConfig"/> for chaining</returns>
         public virtual ClientNetworkConfig SetAddresses(IList<string> addresses)
         {
             _addressList.Clear();
             _addressList.AddRange(addresses);
             return this;
         }
 
+        /// <summary>
+        /// While client is trying to connect initially to one of the members in the configured address list,
+        /// all might be not available. Instead of giving up, throwing Exception and stopping client, it will
+        /// attempt to retry as much as connection Attempt Limit times.
+        /// </summary>
+        /// <param name="connectionAttemptLimit">
+        /// number of times to attempt to connect;
+        /// A zero value means try forever.
+        /// A negative value means default value
+        /// </param>
+        /// <returns><see cref="ClientNetworkConfig"/> for chaining</returns>
         public virtual ClientNetworkConfig SetConnectionAttemptLimit(int connectionAttemptLimit)
         {
             _connectionAttemptLimit = connectionAttemptLimit;
             return this;
         }
 
+        /// <summary>
+        /// Sets connection attempt period in millis, Period for the next attempt to find a member to connect.
```

Comment:
/// Sets the connection attempt period, in millis, for the next attempt to find a member to connect.